src/Entity/Game.php

```diff
@@ -16,39 +16,35 @@ class Game
     #[ORM\Column]
     private ?int $id = null;
 
-    #[ORM\ManyToOne(inversedBy: 'games')]
+    #[ORM\ManyToOne(targetEntity: Room::class, inversedBy: 'games')]
     #[ORM\JoinColumn(nullable: false)]
     private ?Room $room = null;
 
     #[ORM\Column(length: 255, nullable: false)]
     private ?string $word = null;
 
-    #[ORM\Column(type: Types::TIME_MUTABLE)]
-    private ?\DateTimeInterface $duration = null;
+    #[ORM\Column]
+    private ?int $duration = null;
 
-    #[ORM\Column(type: Types::TIME_MUTABLE)]
-    private ?\DateTimeInterface $finished_at = null;
+    #[ORM\Column(nullable: true)]
+    private ?\DateTime $finished_at = null;
```

Comment:
`finished_at` indicates if the game has been finished before `duration`, so it should be an int too

Reply:
👍